.claude/start-frontend-worktree.sh

```
@@ -0,0 +1,4 @@
+#!/bin/sh
+export PATH=/usr/local/bin:/usr/bin:/bin:/usr/sbin:/sbin
+cd /Users/pavelnovak/flow-tasks/.claude/worktrees/eager-burnell/frontend
```

Shellcheck (0.11.0)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)